sdk/core/Azure.Core/src/AzureKeyCredential.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+using System.Threading;
+
+namespace Azure.Core
+{
+    /// <summary>
+    /// API key credential used to authenticate to an Azure Service.
+    /// It provides the ability to update the API key without creating a new client.
+    /// </summary>
+    public class AzureKeyCredential
+    {
+        private string _apiKey;
+
+        /// <summary>
+        /// API key used to authenticate to an Azure service.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string ApiKey
```

Comment:
This property should be called Key not ApiKey

Reply:
Updated